src/routes/safe/components/Balances/SendModal/screens/ReviewSendFundsTx/index.tsx

```diff
@@ -99,7 +99,8 @@ const ReviewSendFundsTx = ({ onClose, onPrev, tx }: ReviewTxProps): React.ReactE
   const submitTx = async (txParameters: TxParameters, delayExecution: boolean) => {
     if (isSpendingLimitTx && txToken && tx.tokenSpendingLimit) {
       const spendingLimitTokenAddress = isSendingNativeToken ? ZERO_ADDRESS : txToken.address
-      const spendingLimit = getSpendingLimitContract()
+      const spendingLimitModuleAddress = getSpendingLimitModuleAddress(_getChainId())
```

Comment:
As this in a component I'd suggest selecting the `chainId` from the store instead.